src/leappto/actor_cli.py

```diff
@@ -0,0 +1,265 @@
+
+
+def _to_port_map(items):
+    port_map = []
+    for target, source in items:
+        port_map.append({
+            'protocol': 'tcp',
+            'exposed_port': int(target),
+            'port': int(source)})
+    return {'ports': port_map}
+
+
+def _path_spec(arg):
+    path = os.path.normpath(arg)
+    if not os.path.isabs(path):
+        raise ValueError("Path '{}' is not absolute or valid.".format(str(arg)))
+
+    return path
+
+
+def _make_base_object(s):
+    return {"value": s}
+
+
+def _migrate_machine_arguments(parser):
+    migrate_cmd = parser.add_parser('migrate-machine', help='migrate source VM to a target container host')
+    migrate_cmd.add_argument("-p", "--print-port-map", default=False, help='List suggested port mapping on target host',
+                             action="store_true")
+    migrate_cmd.add_argument('machine', help='source machine to migrate')
+    migrate_cmd.add_argument('-t', '--target', default='localhost', help='target VM name')
+    migrate_cmd.add_argument(
+        '--tcp-port',
+        default=None,
+        dest="forwarded_tcp_ports",
+        nargs='*',
+        type=_port_spec,
+        help='(Re)define target tcp ports to forward to macrocontainer - [target_port:source_port]'
+    )
+    migrate_cmd.add_argument(
+        '--no-tcp-port',
+        default=None,
+        dest="excluded_tcp_ports",
+        nargs='*',
+        type=_port_spec,
+        help='define tcp ports which will be excluded from the mapped ports [[target_port]:source_port>]'
+    )
+
+    migrate_cmd.add_argument(
+        '--exclude-path',
+        default=None,
+        dest="excluded_paths",
+        nargs='*',
+        type=_path_spec,
+        help='define paths which will be excluded from the source'
+    )
+    migrate_cmd.add_argument("--ignore-default-port-map", default=False,
+                             help='Default port mapping detected by leapp toll will be ignored', action="store_true")
+    migrate_cmd.add_argument('--container-name', '-n', default=None,
+                             help='Name of new container created on target host')
+    migrate_cmd.add_argument(
+        '--force-create',
+        action='store_true',
+        help='force creation of new target container, even if one already exists'
+    )
+    migrate_cmd.add_argument('--disable-start', dest='disable_start', default=False,
+                             help='Migrated container will not be started immediately', action="store_true")
+    migrate_cmd.add_argument('--target-user', default="root", help='Connect as this user to the target via ssh')
+    migrate_cmd.add_argument('--source-user', default="root", help='Connect as this user to the source via ssh')
+    migrate_cmd.add_argument('--debug', default=False, action="store_true", help='Turn on debug logging on stderr')
+
+
+def _migrate_machine(arguments):
+    default_excluded_paths = ['/dev/*', '/proc/*', '/sys/*', '/tmp/*', '/run/*', '/mnt/*', '/media/*', '/lost+found/*']
```

Comment:
i would also add /boot/ here

Reply:
boot is usually not a tempfs or remote

Reply:
it's not but why would you move this to a container?